### ISSUE TITLE
Add in commented parameters when starting a new blog post

### ISSUE THREAD

**Issue by jaredmorgs** [HubPress/hubpress.io#458]
I've started writing posts again, and what I've found is this is my workflow:

1. Click `New Post`.
2. Forget the parameters I need to set always
3. Go back to the post list.
4. Open a post
5. Copy the parameters.
6. Close the post.
7. Repeat step one.

## MVP

What I think we need to do is to add all parameters that HubPress supports into the top of new blog posts, all commented out. That way, a user who is new to the platform will know all the HubPress-specific parameters they can set without having to search the docs. 

```asciidoc
= Your Blog Title
// See https://hubpress.gitbooks.io/hubpress-knowledgebase/content/ for information about the parameters.
// :hp-image: /covers/cover.png
// :published_at: 2019-01-31
// :hp-tags: HubPress, Blog, Open_Source,
// :hp-alt-title: My English Title
```

## Iteration 1

If it is possible as part of the MVP, we should let bloggers click a button in the interface that lets them add this content, and perhaps basic templated AsciiDoc content.

Otherwise, leave this to the next iteration.

**Comment by anthonny** [comment]
@jaredmorgs it will be in the next version
![init-new-post](https://cloud.githubusercontent.com/assets/2006548/22866984/2a9a0f82-f17f-11e6-91b8-020913ac5bb3.gif)

**Comment by jaredmorgs** [comment]
Bug is confirmed fixed in 0.7.0. 👍
